### PR TITLE
Cl 15 styling

### LINKED ISSUE #15: Add Google font, Logo into Sidebar and Change Index.html title, add Favicon
**Is your feature request related to a problem? Please describe.**
Needs app logo and replace template's title 'React Redux App' to app name. Also need to import Google font.

**Describe the solution you'd like**
Add logo in Sidebar and replace `title` tag content, and favicon. Put Google font import in html.

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Change title in html, import Google font, add favicon.
2. Add logo in Sidebar.

## Purpose
Styling and replacing boilerplate code.

## Approach
HTML, styled-components.

Closes #15

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,14 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/pacman.png" />
```
Comment (Lap343) [inline public/index.html:5]: pacman?!?!?!?!

Comment (kmausolf) [reply public/index.html:5]: Omnomnomnom

**Review by Lap343 — APPROVED** [review]
Code looks great can't to check it out

**Review by BlackCubes — APPROVED** [review]
Noice!

**Review by JorgeLVilla — APPROVED** [review]
Looking forward to seeing it!

**Pull request merged**